lib/helpers/CloudFirebaseHelper.dart

```diff
@@ -0,0 +1,54 @@
+//import 'ItemsHelper.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:trovami/model/DocItem.dart';
+
+class CloudFirebaseHelper {
+  Future addItemToCloud(DocItem item, String collectionName) async{
```

Comment:
It's a small thing but maybe we can call these function names a bit more simpler? How about "postItem"

Reply:
Naming is very important to me.  I like your suggestions.
I like to use different verbs for retrieving data from cloud, vs retrieving data from local source.
Post is clearly to the cloud, "Save" or something similar we could use to store locally.
I like to us Fetch for the cloud and Get locally. But I'll go with Get for cloud now.

Reply:
Yeah that's correct but we use `CloudFirebaseHelper().postItem()` right? so the object `CloudFirebaseHelper()` itself tells it is to the cloud. It its a class that store locally we would call it `LocalStoragehelper()` so both cases the object conveys where it's going and the method call tells which action.